script.js

```diff
@@ -0,0 +1,25 @@
+var block = document.getElementById("test");
+var left = 0;
```

Comment:
NO VAR FOREVER AND EVER
use let or const 
learn what is the difference, and how to use them